GolemLib/GolemPrice.cs

```diff
@@ -0,0 +1,71 @@
+namespace GolemLib.Types;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+
+/// <summary>
+/// Represents price settings in Golem pricing model.
+/// TODO: We will find out later which of these options make the most sense.
+/// </summary>
+public class GolemPrice: INotifyPropertyChanged
+{
+
+    private decimal startPrice;
```

Comment:
Why not
`private decimal StartPrice { get; set; }`
and no setter/getter below?
Set would not trigger `OnPropertyChanged`?

Reply:
No, OnPropertyChanged has to be called explicite